### PR TITLE
The first round of tests made by Tyler Shelton

### PR DESCRIPTION
**Pull request by tylersheltonnn**

### CODE REVIEW

File: TS_lobocTests/is1_22.loboc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// ISEQ1: Test addition
```
Comment (johncarl81) [inline TS_lobocTests/is1_22.loboc]: @tylersheltonnn, could you update the comments to reflect the contents of the file?

**Comment by johncarl81** [comment]
@tylersheltonnn, what do you think about moving the tests to the root and keeping the tlc test naming convention `is*.loboc` to avoid the need for the `testRunner.sh` shell script?

File: TS_lobocTests/is1_22.loboc (hunk 1 above)
Comment (tylersheltonnn) [reply TS_lobocTests/is1_22.loboc]: Did it!

**Comment by tylersheltonnn** [comment]
I think that is a good idea.

**Comment by tylersheltonnn** [comment]
I worry  it may make keeping all the tests in order if people push the same named tests.

**Comment by johncarl81** [comment]
The git branching model should make that a non-issue.  I'll make sure it doesn't happen too.

**Comment by tylersheltonnn** [comment]
Okay. I am changing it now.

**Comment by johncarl81** [comment]
By the way, to move your content in the repo, use `git mv from _here/* .`

File: is1_13.loboc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// ISEQ1: Test addition and mu;tiplication based on PEDMAS
```
Comment (johncarl81) [inline is1_13.loboc]: `mu;tiplication` spelling

File: is1_19.loboc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// ISEQ1: Test multiplication with division
+//tlc type: run
+//tlc case:  16=268369920,16711680,0,0,1
+
+//tlc case:  -1=0xffff0000,0x0000ffff,0,0,1
+//tlc case:  1=0xffff0000,0xffff0000,0,0,1
+
```
Comment (johncarl81) [inline is1_19.loboc]: Remove this newline?

File: is1_19.loboc [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// ISEQ1: Test multiplication with division
+//tlc type: run
+//tlc case:  16=268369920,16711680,0,0,1
+
```
Comment (johncarl81) [inline is1_19.loboc]: Remove this newline?

File: is1_34.loboc [hunk 5]
```diff
@@ -0,0 +1,16 @@
+// ISEQ1: Test addition
```
Comment (johncarl81) [inline is1_34.loboc]: Update comment.

**Comment by johncarl81** [comment]
Looks like everything is commented to be under ISEQ1, but I think some of these are under ISEQ2/3.. can you update?

File: is1_31.loboc [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+//tlc case:  f=f,f,t,f 
+//tlc case:  t=t,t,f,f
+//tlc case:  t=t,t,f,f
+//tlc case:  f=t,f,t,f
+//tlc case:  t=t,t,t,f
+//tlc case:  t=f,f,t,t
+
+bool a,b,c,d;
+(a && b) || (c && d);
+
```
Comment (johncarl81) [inline is1_31.loboc]: This is very similar to `is1_30.loboc`, combine?

Comment (tylersheltonnn) [reply is1_31.loboc]: Done. I changed the test.

File: is1_34.loboc (hunk 5 above)
Comment (tylersheltonnn) [reply is1_34.loboc]: Done.

File: is1_19.loboc (hunk 4 above)
Comment (tylersheltonnn) [reply is1_19.loboc]: Done

File: is1_19.loboc (hunk 3 above)
Comment (tylersheltonnn) [reply is1_19.loboc]: Done

File: is1_13.loboc (hunk 2 above)
Comment (tylersheltonnn) [reply is1_13.loboc]: Done

**Comment by johncarl81** [comment]
Looking good @tylersheltonnn, I'll run my codebase against these a little later on today to verify before the official merge.

**Comment by johncarl81** [comment]
Oh, and looks like there are still a few ISEQ comments that need updating.

**Comment by johncarl81** [comment]
Tests pass my compiler and visual inspection looks good.  I'm going to go ahead and merge and make the minor updates I see.

**Pull request merged**